low-level/sspl-ll.spec

```diff
@@ -138,14 +137,11 @@ SSPL_DIR=/opt/seagate/%{product_family}/sspl
 
 # Copy init script
 [ -f $SSPL_DIR/sspl_init ] ||
-    ln -s $SSPL_DIR/bin/sspl_provisioner_init $SSPL_DIR/sspl_init
+    ln -s $SSPL_SETUP/sspl_provisioner_init $SSPL_DIR/sspl_init
 
-# Creating softlink under site-packages to use sspl module easier way.
-PYTHON_BASE_DIR=/usr/lib/python3.6/site-packages/cortx/sspl
+# Creating softlink under sspl/setup to use sspl module easier way.
 [ -d "${SSPL_DIR}" ] && {
-    mkdir -p $PYTHON_BASE_DIR
-    ln -sf $SSPL_DIR/bin $PYTHON_BASE_DIR/bin
-    ln -sf $SSPL_DIR/low-level $PYTHON_BASE_DIR/lowlevel
+    ln -sf $SSPL_DIR/low-level/*/ $SSPL_SETUP/
```

Comment:
You don't need to link low-level to setup dir. Just copy files from setup dir to low-level and then link needed files to /opt/seagate/cortx/sspl/bin

Reply:
If we copy all files from setup to low-level then lots of code will be duplicated and also we need to update too many files that are using setup inside the script.
We can solve this problem with the minimal change that only doing a soft link in setup.
What do you think @sandeepsgt @cdeshmukh ?

Reply:
SSPL_DIR=/opt/seagate/%{product_family}/sspl
SSPL_SETUP=$SSPL_DIR/low-level/files/opt/seagate/sspl/setup

So effectively above statement becomes
 ln -sf /opt/seagate/cortx/sspl/low-level/*/    /opt/seagate/cortx/sspl/low-level/files/opt/seagate/sspl/setup

This is convoluted @satish-seagate 

> Just copy files from setup dir to low-level and then link needed files to /opt/seagate/cortx/sspl/bin

Copy file under setup to bin directly(as happening currently) is not helping right. In that case, above suggestion by @sandeepsgt seems fine.

Reply:
`If we copy all files from setup to low-level then lots of code will be duplicated` Ok. If you feel there will be duplicate files, do `mv` instead of `cp`
`we need to update too many files that are using setup inside the script.` Didn't get this 
`We can solve this problem with the minimal change that only doing a soft link in setup.` This is not right as you are making whole source code available in `$SSPL_DIR/bin`. Functionality wise it will work. But, as the directory suggests , `bin` should have executables.

Reply:
Done